lib/backend/k8s/resources/globalconfig_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package resources_test
+
+import (
+	"github.com/projectcalico/libcalico-go/lib/backend/k8s/resources"
+	"github.com/projectcalico/libcalico-go/lib/backend/k8s/thirdparty"
+	"github.com/projectcalico/libcalico-go/lib/backend/model"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Global Felix config conversion methods", func() {
+
+	converter := resources.GlobalConfigConverter{}
+
+	// Define some useful test data.
+	listIncomplete := model.GlobalConfigListOptions{}
+
+	// Compatible set of list, key and name (used for Key to Name conversion)
+	list1 := model.GlobalConfigListOptions{
+		Name: "AbCd",
+	}
+	key1 := model.GlobalConfigKey{
+		Name: "AbCd",
+	}
+	name1 := "abcd"
+
+	// Compatible set of key and name (Used for Name to Key conversion)
+	key2 := model.GlobalConfigKey{
+		Name: "foobaz",
+	}
+	name2 := "foobaz"
```

Comment:
I think this isn't really the right test - if we wanted to support this it would be `foobaz -> FooBaz`.

Related to my other comment.